notify-debouncer-mini/Cargo.toml

```diff
@@ -19,7 +19,9 @@ name = "notify_debouncer_mini"
 path = "src/lib.rs"
 
 [features]
-default = ["crossbeam-channel"]
+default = ["crossbeam"]
+# can't use dep:crossbeam-channel and feature name crossbeam-channel below rust 1.60
+crossbeam = ["crossbeam-channel","notify/crossbeam-channel"]
```

Comment:
Another way would be renaming our dependency like:

```toml
[dependencies]
crossbeam = { package = "crossbeam-channel", version = "..." }
```

Reply:
But the `crossbeam` feature name doesn't sound that bad, so I'm fine either way :)

Reply:
oh didn't think of renaming them, but yeah I'll rename the feature in the debouncer